Cargo.toml

```diff
@@ -127,18 +127,18 @@ bcrypt = "0.16.0"
 bech32 = "0.11.0"
 bincode = "1.3.3"
 bip39 = "2.1.0"
-bitcoincore-rpc = "0.19.0"
```

Comment:
[nit] What triggered the sorting? Also, in helix if I select and pipe to `sort` I get

```
bitcoincore-rpc = "0.19.0"
bitcoin_hashes = "0.14.0"
bitcoin-io = "0.1.2"
bitcoin-units = "0.1.2"
bitcoin = { version = "0.32.5", features = ["serde"] }
```

Any hunch why there's a difference?

Reply:
I recently was investigating. `sort` considers `-` as a separator and does something ridiculous when it's there. I found no way to change it, so just gave up on sorthing things with `pipe`. I installed https://github.com/tamasfe/taplo and added `.taplo.toml` to our repo already, which is the reason for these modifications.